plugins/by-name/alpha/default.nix

```diff
@@ -156,20 +151,29 @@ in
       opt = options.plugins.alpha;
     in
     lib.mkIf cfg.enable {
-      # TODO: deprecated 2024-08-29 remove after 24.11
-      warnings = lib.mkIf opt.iconsEnabled.isDefined [
+
+      # TODO: added 2024-09-20 remove after 24.11
+      warnings = lib.optionals opt.iconsEnabled.isDefined [
         ''
-          nixvim (plugins.alpha):
           The option definition `plugins.alpha.iconsEnabled' in ${lib.showFiles opt.iconsEnabled.files} has been deprecated; please remove it.
-          You should use `plugins.alpha.iconsPackage' instead.
         ''
       ];
+      plugins.web-devicons =
+        lib.mkIf
+          (
+            opt.iconsEnabled.isDefined
+            && cfg.iconsEnabled
+            && !(
+              config.plugins.mini.enable
+              && config.plugins.mini.modules ? icons
+              && config.plugins.mini.mockDevIcons
+            )
+          )
+          {
+            enable = lib.mkOverride 1490 true;
+          };
```

Comment:
Is there any difference between doing

```nix
plugins.web-devicons = mkIf () { enable = mkOverride; };
```

vs
```nix
plugins.web-devicons.enable = mkIf () (mkOverride);
```

?

I have a feeling the former is better than the latter? No big deal, I just noticed that both techniques have been used in the PR.

Reply:
Shouldn't be, if it was done it wasn't intentional though.